templates/field--field-funding-contact-phone.html.twig

```diff
@@ -0,0 +1,6 @@
+<div class="funding-ct--summary-section--contact-info-phone">
+    {% for item in items %}
+        <span class="wdn-icon-phone" aria-hidden="true"></span><span class="wdn-text-hidden">phone icon</span>{{ item }}
```

Comment:
Is every instance of multiple items a list of items? idk, this doesn't bother me